### DISCUSSION TITLE
Whats the trick for embedded content?

### DISCUSSION THREAD

**Discussion by smegoff** [outline/outline#8001] [General]
Hi All,

When I attempt to - say for example, embedd a typeform link - Outline always says "Sorry, that link won’t work for this embed type"

I couldn't find anything in the documentation referencing a format that's required to make it work.

Cheers
Sean

**Comment by smegoff** [comment]
Ok, figured it out.

![image](https://github.com/user-attachments/assets/e531655d-7c7e-47a1-a38f-259a8fcf4984)

**Reply by tommoor** [reply]
It depends on the service, we probably do need docs covering each individual one here.
